sdk/python/tests/utils/feature_records.py

```diff
@@ -6,7 +6,7 @@
 from pandas.testing import assert_frame_equal as pd_assert_frame_equal
 from pytz import utc
 
-from feast import FeatureStore, utils
+from feast import FeatureService, FeatureStore, utils
```

Comment:
ideally these imports should come from `feast.feature_service`, `feast.feature_store`, etc.

Also,  `utils` :(